pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/bestpractices/xml/ApexUnitTestClassShouldHaveAsserts.xml

```diff
@@ -102,8 +102,8 @@ public class Foo {
         <code><![CDATA[
 @isTest
 public class Foo {
-   public static testMethod void testAssertIsTrue() {
-      Assert.assertEquals(someCondition);
+   public static testMethod void testAssertEqual() {
+      Assert.assertEquals(null, someCondition);
```

Comment:
I hate to be so picky, but this should probably be `Assert.areEqual(value1, value2)` since `areEqual` is the actual method name in the new `Assert` class and `someCondition` as an argument implies an expression that evaluates to a Boolean value such as would be used by `Assert.isTrue/False()`.

Reply:
Since now `Assert` is a standard Apex class, we should probably change this test and the test before, which explicitly test custom assert classes via the property "additionalAssertMethodPattern".

I'd suggest to just name the class `MyAssert` in these test cases to avoid name clashes:

```java
@isTest
public class Foo {
   public static testMethod void testAssertIsTrue() {
      MyAssert.isTrue(someCondition);
}
```